modules/serverless-logic-deploying-workflows-dev-mode.adoc

```diff
@@ -0,0 +1,137 @@
+// Module included in the following assemblies:
+//
+// * serverless/serverless-logic/serverless-logic-creating-managing-workflows.adoc
+
+:_content-type: PROCEDURE
+[id="serverless-logic-deploying-workflows-dev-mode"]
+= Deploying workflows in Dev mode
+
+You can deploy your local workflow on OpenShift in Dev mode. This allows you to experiment and modify your workflow directly on the cluster, seeing changes almost immediately. Dev mode is designed for development and testing purposes. It is ideal for initial development stages and testing new changes.
+
+.Prerequisites
+
+* {ServerlessLogicOperatorName} is installed on your cluster.
+* You have created a {ServerlessLogicProductName} project or have access to a project with the appropriate roles and permissions to create applications and other workloads in OpenShift Container Platform.
+* You have created your {ServerlessLogicProductName} project
+* Install the OpenShift CLI `(oc)`.
+
+.Procedure
+
+. Create the workflow configuration `YAML` file:
++
+For example, create a sample `workflow-dev.yaml` file containing the following sample code:
++
+[source,yaml]
+----
+apiVersion: sonataflow.org/v1alpha08
+kind: SonataFlow
+metadata:
+  name: greeting
+  annotations:
+    sonataflow.org/description: Greeting example on k8s!
+    sonataflow.org/version: 0.0.1
+    sonataflow.org/profile: dev 
+spec:
+  flow: 
+    start: ChooseOnLanguage
+    functions:
+      - name: greetFunction
+        type: custom
+        operation: sysout
+    states:
+      - name: ChooseOnLanguage
+        type: switch
+        dataConditions:
+          - condition: "${ .language == \"English\" }"
+            transition: GreetInEnglish
+          - condition: "${ .language == \"Spanish\" }"
+            transition: GreetInSpanish
+        defaultCondition: GreetInEnglish
+      - name: GreetInEnglish
+        type: inject
+        data:
+          greeting: "Hello from JSON Workflow, "
+        transition: GreetPerson
+      - name: GreetInSpanish
+        type: inject
+        data:
+          greeting: "Saludos desde JSON Workflow, "
+        transition: GreetPerson
+      - name: GreetPerson
+        type: operation
+        actions:
+          - name: greetAction
+            functionRef:
+              refName: greetFunction
+              arguments:
+                message:  ".greeting + .name"
+        end: true
+----
+
+. Navigate to the directory where the `YAML` file is contained, and deploy the application by applying the `YAML` file:
++
+[source,terminal]
+----
+$ oc apply -f <filename> -n <your_namespace>
+----
++
+.Example command
+[source,terminal]
+----
+$ oc apply -f workflow-dev.yaml
+----
+
+. Verify the deployment and check the status of the the deployed workflow:
++
+[source,terminal]
+----
+$ oc get workflow -n <your_namespace> -w
+----
++
+.Example command
+[source,terminal]
+----
+$ oc get sonataflows
```

Comment:
The example does not fit the generic syntax, there is no -n nor -w

Reply:
fixed it